pylint/lint.py

```diff
@@ -89,6 +90,24 @@
 MANAGER = astroid.MANAGER
 
 
+def _ast_from_string(data, filepath, modname):
+    if (
```

Comment:
Rather than doing three lookups, let's just do one:
```suggestion
    cached = MANAGER.astroid_cache.get(modname)
    if cached and cached.file == filepath:
         return cached
```